core/src/main/java/nu/yona/server/rest/Config.java

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Stichting Yona Foundation
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v.2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server.rest;
+
+import java.util.Collections;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import nu.yona.server.ThreadScope;
+
+@Configuration
+public class Config implements WebMvcConfigurer
```

Comment:
It's a configuration (as indicated by the ``@Configuration`` annotation) with REST-related stuff (and that's indicated because it is in the ``nu.yona.server.rest`` package). Naming it ``WebMvcConfig`` would indicate a more narrow scope than that.